package.json

```diff
@@ -62,24 +62,24 @@
 	"devDependencies": {
 		"@babel/plugin-proposal-private-property-in-object": "^7.21.11",
 		"@babel/preset-react": "^7.26.3",
-		"@babel/preset-typescript": "^7.26.0",
+		"@babel/preset-typescript": "^7.27.0",
 		"@bahmutov/cy-rollup": "^2.0.0",
 		"@cfaester/enzyme-adapter-react-18": "^0.8.0",
 		"@cypress/react18": "^2.0.1",
 		"@rollup/plugin-babel": "^6.0.4",
 		"@rollup/plugin-commonjs": "^28.0.3",
 		"@rollup/plugin-url": "^8.0.2",
-		"@storybook/addon-actions": "^8.6.4",
-		"@storybook/addon-docs": "^8.6.4",
-		"@storybook/addon-essentials": "^8.6.4",
-		"@storybook/addon-links": "^8.6.4",
+		"@storybook/addon-actions": "^8.6.9",
+		"@storybook/addon-docs": "^8.6.9",
+		"@storybook/addon-essentials": "^8.6.9",
+		"@storybook/addon-links": "^8.6.9",
 		"@storybook/addons": "^7.6.17",
-		"@storybook/node-logger": "^8.6.4",
-		"@storybook/react": "^8.6.4",
-		"@storybook/react-webpack5": "^8.6.4",
-		"@storybook/test": "^8.6.4",
+		"@storybook/node-logger": "^8.6.9",
+		"@storybook/react": "^8.6.9",
+		"@storybook/react-webpack5": "^8.6.9",
+		"@storybook/test": "^8.6.9",
 		"@storybook/testing-react": "^2.0.1",
-		"@storybook/theming": "^8.6.4",
+		"@storybook/theming": "^8.6.9",
 		"@svgr/rollup": "^8.1.0",
```

Comment:
We should try to remove this dependency too, there are some references left in the rollup config, but it seems to be no longer used.